src/cms/forms/regions/region_form.py

```diff
@@ -132,7 +132,7 @@ def clean(self):
         if apps.get_app_config("gvz_api").api_available:
             gvz_region = GvzRegion(
                 region_name=cleaned_data["name"],
-                region_key=cleaned_data["common_id"],
+                region_ags=cleaned_data["common_id"],
                 region_type=cleaned_data["administrative_division"],
             )
             if gvz_region.aliases and cleaned_data["aliases"] == "":
```

Comment:
```suggestion
            if gvz_region.aliases and cleaned_data["aliases"] == in ["", "{}"]:
```
An emty JSON encoded as string is `{}`, so I suggest to update the aliases in this case as well.